src/Illuminate/Foundation/Testing/Constraints/SeeInOrder.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Illuminate\Foundation\Testing\Constraints;
+
+use PHPUnit\Framework\Constraint\Constraint;
+
+class SeeInOrder extends Constraint
+{
+    /**
+     * The value that failed. Used to display in the error message.
+     *
+     * @var string
+     */
+    protected $failedValue;
+
+    /**
+     * The string we want to check the content of.
+     *
+     * @var
```

Comment:
Missing `var` type.

Strange that Style CI didn't cached it :thinking:

Reply:
It's not a code style bug, that's why. It's just a syntax error.